main.go

```diff
@@ -91,6 +90,5 @@ func main() {
 		os.Exit(constants.GeneralErrorExitCode)
 	}
 
-	fmt.Printf("\nBuild duration: %dms\n", int64(duration)/int64(time.Millisecond))
```

Comment:
NB: before merging, we should append a "Build succeeded" line; previously the build duration more-or-less served this purpose.